politeiawww/api/v1/v1.go

```diff
@@ -331,10 +331,12 @@ type Login struct {
 
 // LoginReply is used to reply to the Login command.
 type LoginReply struct {
-	IsAdmin   bool   `json:"isadmin"`   // Set if user is an admin
-	UserID    string `json:"userid"`    // User id
-	Email     string `json:"email"`     // User email
-	PublicKey string `json:"publickey"` // Active public key
+	IsAdmin        bool    `json:"isadmin"`        // Set if user is an admin
```

Comment:
At this point, I think we should make LoginReply and MeReply the same structure and document that.  Let's go with LoginReply as the one source of truth.  So Me will start replying with LoginReply and make this obvious in the docs.